idunn/places/pj_poi.py

```diff
@@ -9,15 +9,11 @@
 from ..api.constants import PoiSource
 from ..api.urlsolver import resolve_url
 
-WHEELCHAIN_ACCESSIBLE = re.compile(
-    "|".join(
-        [
-            "accès handicapés?",
-            "accès aux personnes à mobilité réduite",
-            "accès pour personne à mobilité réduite",
-        ]
-    )
-)
+
+CLICK_AND_COLLECT = re.compile(r"retrait .*")
+DELIVERY = re.compile(r"commande en ligne|livraison.*")
+TAKEAWAY = re.compile(r".* à emporter")
+WHEELCHAIN_ACCESSIBLE = re.compile("accès (handicapés?|(aux|pour) personnes? à mobilité réduite)")
```

Comment:
Let's take advantage of this change to fix the typo
```suggestion
WHEELCHAIR_ACCESSIBLE = re.compile("accès (handicapés?|(aux|pour) personnes? à mobilité réduite)")
```